turbo.json

```diff
@@ -23,6 +28,7 @@
       "inputs": ["src/**/*.tsx", "src/**/*.ts"]
     },
     "dev": {
+      "dependsOn": ["//#dev:link"],
```

Comment:
what does that mean? Can you please point me to some documentation to understand that part a little bit more?

Reply:
this means that the `dev` task, which runs the example apps in development mode, is dependant upon the `dev:link` task being run.

docs are here: https://turbo.build/repo/docs/reference/configuration#dependson